lib/index.js

```diff
@@ -85,14 +85,12 @@ Grid.prototype.collection = function (name) {
  */
 
 Grid.prototype.remove = function (options, callback) {
-  var _id;
   if (options._id) {
-    _id = this.tryParseObjectId(options._id) || options._id;
+    options._id = this.tryParseObjectId(options._id) || options._id;
```

Comment:
You are now mutating the `options` object passed in, this is considered a bad practice and can lead to some hard to track down bugs...